shared/src/commonMain/graphql/Queries.graphql

```diff
@@ -20,6 +20,20 @@ query GetConferenceData($first: Int! = 1000, $after: String = null) {
     }
 }
 
+query GetSessions($first: Int! = 1000, $after: String = null) {
```

Comment:
Seems a shame we can't query for session on a day.  Should I change the query?

Reply:
The API doesn't allow this. Can you open an issue? I'll add this.

Reply:
Feel free to specify how you'd want this to work i.e:

* `sessions(after: "2023-04-13-00:00", before: "2023-04-14-00:00")`
* `sessions(day: "2023-04-13")`
* something else?